src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const mirrorBot = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (newKey in mirrorBot) {
+      return null;
```

Comment:
From CHECKLIST 
GOOD EXAMPLE:

`if (key in robot) {`
EVEN BETTER EXAMPLE:

`if (robot.hasOwnProperty(key)) {`